### DISCUSSION TITLE
Install.exe not working (2)

### DISCUSSION THREAD

**Discussion by kendover** [ppy/osu#27046] [Q&A]
### Type

Crash to desktop

### Bug description

similar issue to [f1refa11](https://github.com/f1refa11) from https://github.com/ppy/osu/issues/20868setup but i can't seem to find the file location for this (unless it's referring to "...Users/AppData/Local/SquirrelTemp/SquirrelySetup.log") to help get my issue solved quicker #:sob: 

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/158758777/c950980c-8600-49fd-917c-e8fa2d0c2b2f)

### Version

2024.131.0

### Logs

![image](https://github.com/ppy/osu/assets/158758777/885949f8-7a07-4a63-a561-1729d701874e)

**Comment by bdach** [comment]
please attach the log file you screenshotted there

**Comment by kendover** [comment]
> please attach the log file you screenshotted there

[SquirrelSetup.log](https://github.com/ppy/osu/files/14155251/SquirrelSetup.log)

**Reply by bdach** [reply]
Nothing relevant there.

Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Reply by kendover** [reply]
seems like it doesn't want to upload via github as its own file so i put it on a gdrive folder. Hopefully, it still works.

https://drive.google.com/drive/folders/1p4VgiZRDi9DJ9UvZKpm7Z4WSa9hlH2oc?usp=sharing

**Reply by kendover** [reply]
@bdach what's the current situation to this issue? I would like to know :DD

**Reply by bdach** [reply]
I'm honestly not sure because none of the logs that you've attached thus far show anything approaching a usable error to work with. I'm not sure the squirrel log file was attached from the correct directory, and the event viewer file has just no trace of squirrel.

That said I see errors from _another_ installer (bluestacks) that talk about "unauthorised access". Maybe this is a permission issue on your end. Something about some directories not being accessible / writable.
